vendor/vendor.json

```diff
@@ -114,6 +114,196 @@
+			"path": "github.com/aws/aws-sdk-go-v2/aws/stscreds",
+			"revision": "d52522b5f4b95591ff6528d7c54923951aadf099",
+			"revisionTime": "2018-09-27T22:51:20Z"
+		},
+		{
+			"checksumSHA1": "e6B1TByeUglBqKy8uBXBaINynlE=",
+			"path": "github.com/aws/aws-sdk-go-v2/internal/awsutil",
+			"revision": "d52522b5f4b95591ff6528d7c54923951aadf099",
+			"revisionTime": "2018-09-27T22:51:20Z"
+		},
+		{
+			"checksumSHA1": "ERU2v7Jbu86xjpLgb+dG4lnyNfU=",
+			"path": "github.com/aws/aws-sdk-go-v2/internal/sdk",
+			"revision": "d52522b5f4b95591ff6528d7c54923951aadf099",
+			"revisionTime": "2018-09-27T22:51:20Z"
+		},
+		{
+			"checksumSHA1": "Yfzf9cCDNY/kAG1j7WQx2ZBU7zE=",
+			"path": "github.com/aws/aws-sdk-go-v2/private/protocol",
+			"revision": "d52522b5f4b95591ff6528d7c54923951aadf099",
+			"revisionTime": "2018-09-27T22:51:20Z"
+		},
+		{
+			"checksumSHA1": "FdZh+IZa1ty76+RV7tXVfmtyIuo=",
+			"path": "github.com/aws/aws-sdk-go-v2/private/protocol/json",
+			"revision": "d52522b5f4b95591ff6528d7c54923951aadf099",
+			"revisionTime": "2018-09-27T22:51:20Z"
+		},
+		{
+			"checksumSHA1": "W20qSroKjq+SVZCQWhdOxTVzMJc=",
+			"path": "github.com/aws/aws-sdk-go-v2/private/protocol/json/jsonutil",
+			"revision": "d52522b5f4b95591ff6528d7c54923951aadf099",
+			"revisionTime": "2018-09-27T22:51:20Z"
+		},
+		{
+			"checksumSHA1": "QBgP8y0RJvyqYn234+0t/TsTV38=",
+			"path": "github.com/aws/aws-sdk-go-v2/private/protocol/jsonrpc",
+			"revision": "d52522b5f4b95591ff6528d7c54923951aadf099",
+			"revisionTime": "2018-09-27T22:51:20Z"
+		},
+		{
+			"checksumSHA1": "aHSao8oK9aBHiVerbVN7Ph5NlMU=",
+			"path": "github.com/aws/aws-sdk-go-v2/private/protocol/query",
+			"revision": "d52522b5f4b95591ff6528d7c54923951aadf099",
+			"revisionTime": "2018-09-27T22:51:20Z"
+		},
+		{
+			"checksumSHA1": "4deNSswzG34mHBmXucHZ1HuB/4o=",
+			"path": "github.com/aws/aws-sdk-go-v2/private/protocol/query/queryutil",
+			"revision": "d52522b5f4b95591ff6528d7c54923951aadf099",
+			"revisionTime": "2018-09-27T22:51:20Z"
+		},
+		{
+			"checksumSHA1": "ePhzbmlr/QdIaq/p5yS1YEWJHPY=",
+			"path": "github.com/aws/aws-sdk-go-v2/private/protocol/rest",
+			"revision": "d52522b5f4b95591ff6528d7c54923951aadf099",
+			"revisionTime": "2018-09-27T22:51:20Z"
+		},
+		{
+			"checksumSHA1": "+NM7uyBRkkB2QDMvJnNSGsBFbXQ=",
+			"path": "github.com/aws/aws-sdk-go-v2/private/protocol/restjson",
+			"revision": "d52522b5f4b95591ff6528d7c54923951aadf099",
+			"revisionTime": "2018-09-27T22:51:20Z"
+		},
+		{
+			"checksumSHA1": "aUgbxpCz2vi9bxm+YHnz3GojvAY=",
+			"path": "github.com/aws/aws-sdk-go-v2/private/protocol/xml/xmlutil",
+			"revision": "d52522b5f4b95591ff6528d7c54923951aadf099",
+			"revisionTime": "2018-09-27T22:51:20Z"
+		},
+		{
+			"checksumSHA1": "ZK1OLl3ATOAHEq0LLmmcQ0XH5NI=",
+			"path": "github.com/aws/aws-sdk-go-v2/service/cloudwatchlogs/",
+			"revision": "ff1a530c31507c97cf5edbee226e604ca08661cc",
+			"revisionTime": "2018-05-26T01:14:17Z",
+			"version": "v2.0.0-preview.4",
```

Comment:
Why not all `github.com/aws/aws-sdk-go-v2` have the same version / commit id?

Reply:
Weird, I did all of them at the same time. I will take a second look.